src/lib/utils/fetchGitHubContributors.ts

```diff
@@ -0,0 +1,42 @@
+import { join } from "path"
+import { GITHUB_AUTH_HEADERS, GITHUB_COMMITS_URL } from "@/lib/constants"
+import type { Author } from "@/lib/interfaces"
+import type { FileContributorsState } from "@/lib/types"
+
+export const fetchGitHubAuthors = async (
```

Comment:
To be consistent with the filename
```suggestion
export const fetchGitHubContributors = async (
```

Reply:
Yep, names are a mess... I'll clean them up. The GitHub API uses "Author" terminology, but we really use "Contributor", so I'll convert to that and also clean up the file names to make sure they match the component names.